README.md

```diff
@@ -1,41 +1,42 @@
-# 🚧 Endo 🚧 
+#  Endo
 
 [![build status][ci-svg]][ci-url]
 [![contributing][contributing-svg]][contributing-url]
 [![license][license-image]][license-url]
 
-Endo is a JavaScript platform 👷‍♀️*under development*🏗 for secure communication
-among objects within one process and distributed between mutually suspicious
+Endo is a JavaScript platform under development for secure communication among
+objects within one process and distributed between mutually suspicious
 machines.
-The foundation of Endo is [SES][], a tamper-proof JavaScript environment that
-allows safe execution of arbitrary programs in Compartments.
-
-⚠️Endo is not Done. All following statements about Endo are aspirational.⚠️
 
+The foundation of Endo is Hardened JavaScript as implemented by [`ses`][SES], a
+tamper-proof JavaScript environment that allows safe execution of arbitrary
+programs in Compartments.
 Most JavaScript libraries built for Node.js, either in CommonJS or ECMAScript
 module format, are suitable for running in Endo without modification, since
 such programs rarely tamper with global scope or shared intrinsic objects.
 The exception is shims, which require special treatment and express consent to
 work with Endo.
 
 Endo protects program integrity both in-process and in distributed systems.
-SES protects local integrity, defending an application against [supply chain
-attacks][]: hacks that enter through upgrades to third-party dependencies.
+Hardened JavaScript protects local integrity, defending an application against
+[supply chain attacks][]: hacks that enter through upgrades to third-party
+dependencies.
```

Comment:
For bonus points, work these in somewhere to explain capabilities and POLA:

In 3 minutes:

 - [Opening Statement on SOSP 50th Anniversary Panel](https://www.youtube.com/watch?v=br9DwtjqmVI) Mark Miller

In 15 minutes:

 - [Navigating the Attack Surface to achieve a multiplicative reduction in risk](https://www.youtube.com/watch?v=wW9-KuezPp8&t=664s)